### DISCUSSION TITLE
Error: [html-rspack-plugin]: Child compilation failed:

### DISCUSSION THREAD

**Discussion by ayangweb** [web-infra-dev/rspress#1593] [Q&A] [answered]
When deploying to cloudflare, an error is reported!

![Image](https://github.com/user-attachments/assets/757d4d74-2620-4583-af96-f7a2506f16d4)

**Comment by chenjiahan** [comment, marked as answer]
Related to: https://github.com/web-infra-dev/rsbuild/issues/3994

**Comment by chenjiahan** [comment]
Fixed in Rspack v1.1.2
